storyStarterGame.js

```diff
@@ -0,0 +1,470 @@
+const readLineSync = require('readline-sync');
+
+    
+const welcome = () => {
+    let nameInput = readLineSync.question("What is your name?") // Ask the player what is their name. 
+
+console.log (`Hello ${nameInput}!  It's nice to meet you.`) // Using the information gotten above, this line prints a greeting with the player's name 
+console.log ("Welcome to Story Generator Game!") // Welcoming the player to the game
+    console.log ("Where I will help you write the next best seller!") // Stating the purpose of the game
+    if (readLineSync.keyInYN("Are you ready to begin?")) { // This line ask asks the player if they are ready to begin the game. 
+        console.clear() // This clears all of the previous lines 
+        startGame(); // If the player presses Y then the game will go straight to this function, starting the game!
+    } else {
+        leaveGame(); // If the player presses N then the game will go straight to this function, ending the game!
+    }
+}
+
+
+    // Starting Story Game 
+const startGame = () => {
+    console.log ("There are two genres we will be working with today") // Letting the player know that there are going to be two chocies here. 
+    let choice = readLineSync.question("The choices are Romance and Suspense! Which would you like to choose to be your genre?") // Asking whether or not the player wants a romance story or a suspense story
+    if (choice === "Suspense" || choice === "suspense") { // if the choice that was made equal to the input "Suspense" 
+        console.clear(); // Clears all of the previous lines 
+        suspenseStory(); // If player's input equals to "suspense" than it will run into the next function.
+    } else if (choice === "Romance" || choice === "romance"){ // if the choice that was made equal to "romance"
```

Comment:
try doing the same here with the `.toLowerCase()`

Reply:
Got it. I believe I know how to do it now.